CHANGELOG.rst

```diff
@@ -17,6 +17,10 @@ Unreleased
 ----------
 * nothing unreleased
 
+[4.19.8]
+--------
+* chore: updated permission class ``IsInProvisioningAdminGroup`` for ``PendingEnterpriseCustomerAdminUser`` viewpoint and handled exceptions in serializer.
```

Comment:
this isn't a chore commit.

Reply:
@sameenfatima78 feedback was to avoid using `feat` for this.